src/roles/common/tasks/check-predeploy-prereq.yml

```diff
@@ -179,3 +179,21 @@
   fail:
     msg: "We couldn't find the user's SSH key file {{ user_ssh_pub_key }}. Is user_ssh_pub_key defined correctly in common.yml?"
   when: target_server_type | default('NONE') is not match ("openstack") and current_user_ssh_key.rc != 0
+
+- name: Check if vsd_license_file is present
+  include_role:
+    name: common
+    tasks_from: check-local-file-present
+  vars:
+    file_path: "{{ vsd_license_file }}"
+    success_message: "Found VSD license file"
+    fail_message: "Could not find VSD license file"
```

Comment:
This needs to evaluated in context with some of the recent changes that went in to not fail if the VSD license file is present. Can you check with @urana-nuage if the skip license check condition should be used here? 

Additionally, does this fail if the vsd_license_file variable is present, but license file cannot be. found. Or does it fail if vsd_license_file variable is not present?

Reply:
okay, I'll check with him and add another reply once I get an update

Reply:
I'm using the stat module to check if the license file exists locally, and if it doesn't, the assertion will fail, which will cause a failure of the test run.

If the vsd_license_file is present, but the license file cannot be found, I'm not sure what will happen. I've assumed that if the file is present locally, then it can be found by the stat module. Is there a particular situation you have in mind that the file would be present, but cannot be found by the stat module? If there is, I'll probably have to add something to account for that.

If you just want an error message to be print out, without the test immediately failing, I'm assuming I can just add an ignore_errors: true to the end of the "Assert that VSD license file must exist" task in check-local-present.yml